src/test/java/code/practice/tasks/FilesTaskTest.java

```diff
@@ -0,0 +1,94 @@
+package code.practice.tasks;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class FilesTaskTest {
+    private final String VALID_DATA_FILE_URI = "./src/test/resources/FilesTaskTest_1/ValidDataFile.txt";
+    private final String EMPTY_FILE_URI = "./src/test/resources/FilesTaskTest_1/EmptyFile.txt";
+    private final String ONLY_LETTERS_FILE_URI = "./src/test/resources/FilesTaskTest_1/OnlyLettersFile.txt";
+    private final String VALID_DATA_FILE_PROTECTED_URI =
+            "./src/test/resources/FilesTaskTest_1/ValidDataFileProtected.txt";
+    private final String FILE_NOT_FOUND_URI = "./src/test/resources/" +
+            "FilesTaskTest_1/FileIsNotFound.txt";
+    private final String INVALID_PATH_URI = "./src/test/resources/FilesTaskTest_1?/ValidDataFile.txt";
+    private final String INVALID_DIRECTORY_URI = "./src/test/resources/FilesTaskTest/ValidDataFile.txt";
+
+    private final FilesTask filesTask = new FilesTask();
+
+    @BeforeEach
+    void setUp() throws IOException {
+        Files.writeString(Path.of(VALID_DATA_FILE_URI), "1,2,3,4,5,6,7,8,9");
```

Comment:
This won't be executed if file is changed in the last test method. Also we don't need this for each test method, please create method `revert(File file, String content) {}` and execute it in tests where file is changed. Don't forget that it shoud be executed even if exception happened during test execution.